doc/classes/Translation.xml

```diff
@@ -99,5 +121,8 @@
 		<member name="locale" type="String" setter="set_locale" getter="get_locale" default="&quot;en&quot;">
 			The locale of the translation.
 		</member>
+		<member name="plural_rule" type="String" setter="set_plural_rule" getter="get_plural_rule" default="&quot;&quot;">
+			The plural forms rule string.
```

Comment:
Could _at least_ use a few noteworthy examples.

Reply:
Added link to the `gettext` docs. Since there are now default rules, this is less relevant and only useful for custom languages.